BBus/BBus/BusRoute/View/BusStationTableViewCell.swift

```diff
@@ -0,0 +1,133 @@
+    
+    var titleLeadingOffset: CGFloat { return 107 }
+    var lineTrailingOffset: CGFloat { return -20 }
+    
+    private lazy var beforeLineView = UIView()
+    private lazy var afterLineView = UIView()
+    private lazy var centerImageView = UIImageView()
+    
+    private lazy var stationTitleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 17, weight: .semibold)
+        return label
+    }()
+    
+    private lazy var stationDescriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.textColor = BusRouteViewController.Color.tableViewCellSubTitle
+        return label
+    }()
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+
+        self.configureLayout()
+        self.selectionStyle = .none
+    }
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        self.configureLayout()
+        self.selectionStyle = .none
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+
+        self.stationTitleLabel.text = ""
+        self.stationDescriptionLabel.text = ""
+    }
+
+    // MARK: - Configure
+    private func configureLayout() {
+        self.addSubview(self.stationTitleLabel)
+        self.stationTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            self.stationTitleLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 18),
+            self.stationTitleLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: titleLeadingOffset),
+            self.stationTitleLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -20)
+        ])
+        
+        self.addSubview(self.stationDescriptionLabel)
+        self.stationDescriptionLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            self.stationDescriptionLabel.topAnchor.constraint(equalTo: self.stationTitleLabel.bottomAnchor, constant: 5),
+            self.stationDescriptionLabel.leadingAnchor.constraint(equalTo: self.stationTitleLabel.leadingAnchor),
+            self.stationDescriptionLabel.trailingAnchor.constraint(equalTo: self.stationTitleLabel.trailingAnchor)
+        ])
+        
+        self.addSubview(self.beforeLineView)
+        self.beforeLineView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            self.beforeLineView.heightAnchor.constraint(equalTo: self.heightAnchor, multiplier: 0.5),
+            self.beforeLineView.widthAnchor.constraint(equalToConstant: 3),
+            self.beforeLineView.topAnchor.constraint(equalTo: self.topAnchor),
+            self.beforeLineView.centerXAnchor.constraint(equalTo: self.stationTitleLabel.leadingAnchor, constant: lineTrailingOffset)
+        ])
```

Comment:
```suggestion
        NSLayoutConstraint.activate([
            self.beforeLineView.heightAnchor.constraint(equalTo: self.heightAnchor, multiplier: 0.5),
            self.beforeLineView.widthAnchor.constraint(equalToConstant: 3),
            self.beforeLineView.topAnchor.constraint(equalTo: self.topAnchor),
            self.beforeLineView.centerXAnchor.constraint(equalTo: self.stationTitleLabel.leadingAnchor, constant: self.lineTrailingOffset)
        ])
```

`self`를 붙여줄 수 있을 것 같아요!